### PR TITLE
Created REF indicators and added schema

### PR DESCRIPTION
**Pull request by Juan-Mateos**

### CODE REVIEW

File: data/processed/ref/2020_02_04_mean_ref.csv [hunk 1]
```diff
@@ -0,0 +1,41 @@
+,year,nuts_id,nuts_year_spec,mean_ref
```
Comment (mindrones) [inline data/processed/ref/2020_02_04_mean_ref.csv]: I would remove the index column (which has no header anyway)

File: data/processed/ref/2020_02_04_mean_ref_stem.csv [hunk 2]
```diff
@@ -0,0 +1,39 @@
+,year,nuts_id,nuts_year_spec,mean_ref_stem
```
Comment (mindrones) [inline data/processed/ref/2020_02_04_mean_ref_stem.csv]: I would remove the index column here too

File: data/processed/ref/2020_02_04_total_4_fte.csv [hunk 3]
```diff
@@ -0,0 +1,41 @@
+,year,nuts_id,nuts_year_spec,total_4_fte
```
Comment (mindrones) [inline data/processed/ref/2020_02_04_total_4_fte.csv]: I would remove the index column here too

File: data/processed/ref/2020_02_04_mean_ref.csv (hunk 1 above)
Comment (mindrones) [reply data/processed/ref/2020_02_04_mean_ref.csv]: Also I'd rename the file to just `mean_ref.csv` and I'd add a `data_date` field in the schema like:

```yaml
data_date: 20200204
```

File: data/processed/ref/2020_02_04_mean_ref_stem.csv (hunk 2 above)
Comment (mindrones) [reply data/processed/ref/2020_02_04_mean_ref_stem.csv]: I'd move the date to the schema here too

File: data/processed/ref/2020_02_04_total_4_fte.csv (hunk 3 above)
Comment (mindrones) [reply data/processed/ref/2020_02_04_total_4_fte.csv]: I'd move the date to the schema here too

File: data/processed/ref/mean_ref.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+api: https://results.ref.ac.uk
+api_documentation: https://www.ref.ac.uk/2014/pubs/201401/
+description: These are the results of the Research Excellence Framework, where university departments in various disciplines are assessed on the quality of their research. The latest REF was conducted in 2014 so data is available only for one year
+endpoint_source: https://results.ref.ac.uk/(S(hlvnuqzwkag44jp3df3d4q14))/DownloadFile/AllResults/xlsx
+enpoint_static: N/A
```
Comment (mindrones) [inline data/processed/ref/mean_ref.yaml]: If a field is undefined I'd omit it?

File: data/processed/ref/mean_ref.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+framework_group: 1
+is_experimental: 0
```
Comment (mindrones) [inline data/processed/ref/mean_ref.yaml]: Although I know Python booleans internally are integers, I'd prefer to see `True` or `False` here.

File: data/processed/ref/mean_ref_stem.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+api: https://results.ref.ac.uk
+api_documentation: https://www.ref.ac.uk/2014/pubs/201401/
+description: These are the results of the Research Excellence Framework, where university departments in various disciplines are assessed on the quality of their research. The latest REF was conducted in 2014 so data is available only for one year
+endpoint_source: https://results.ref.ac.uk/(S(hlvnuqzwkag44jp3df3d4q14))/DownloadFile/AllResults/xlsx
+enpoint_static: N/A
```
Comment (mindrones) [inline data/processed/ref/mean_ref_stem.yaml]: same here

File: data/processed/ref/mean_ref_stem.yaml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+framework_group: 1
+is_experimental: 0
```
Comment (mindrones) [inline data/processed/ref/mean_ref_stem.yaml]: same here

File: data/processed/ref/total_4_fte.yaml [hunk 8]
```diff
@@ -0,0 +1,23 @@
+api: https://results.ref.ac.uk
+api_documentation: https://www.ref.ac.uk/2014/pubs/201401/
+description: These are the results of the Research Excellence Framework, where university departments in various disciplines are assessed on the quality of their research. The latest REF was conducted in 2014 so data is available only for one year
+endpoint_source: https://results.ref.ac.uk/(S(hlvnuqzwkag44jp3df3d4q14))/DownloadFile/AllResults/xlsx
+enpoint_static: N/A
```
Comment (mindrones) [inline data/processed/ref/total_4_fte.yaml]: same here

File: data/processed/ref/total_4_fte.yaml [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+framework_group: 1
+is_experimental: 0
```
Comment (mindrones) [inline data/processed/ref/total_4_fte.yaml]: same here

**Review by mindrones — CHANGES_REQUESTED** [review]
Some suggested changes, see below

File: data/processed/ref/2020_02_04_mean_ref.csv (hunk 1 above)
Comment (Juan-Mateos) [reply data/processed/ref/2020_02_04_mean_ref.csv]: Thanks Luca! I will address these later :-)

**Comment by mindrones** [comment]
Forgot to add, if we add a `data_date` field to the schema, we must also add a definition in `data/schema/schema_template.yaml`:

```yaml
data_date: Datestring
```

and in `data/schema/types_schema.yaml` we must define the format of a date string, like:

```yaml
Datestring:
  date_type: string
  format: YYYYMMDD
  kind: date
```

**Comment by Juan-Mateos** [comment]
Hi @mindrones I pulled a new version addressing your requests. Let me know what you think!

**Pull request closed without merging**